introduction.md

```diff
@@ -0,0 +1,7 @@
+Hi, my name is Alicia, I'm an IT Developer at the University of Wisconsin.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.